pkg/volume/plugins.go

```diff
@@ -183,9 +183,6 @@ type VolumeHost interface {
 	// directory might not actually exist on disk yet.
 	GetPodPluginDir(podUID types.UID, pluginName string) string
 
-	// GetKubeClient returns a client interface
```

Comment:
instead of directly exposing client and cloud provider and refactoring all plugins' `Init()`, would you add a constructor to initialize a `VolumeHost` with `client` and `cloudprovider`? Then call this constructor in `InitPlugins` and get a `VolumeHost`, then we should be able to reuse `plugin.Init(host)`